content/influxdb/cloud-dedicated/admin/databases/create.md

```diff
@@ -104,7 +124,174 @@ The retention period value cannot be negative or contain whitespace.
 {{% /flex-content %}}
 {{< /flex >}}
 
-## Database naming restrictions
+### Custom partitioning (influxctl CLI)
+
+{{< product-name >}} lets you define a custom partitioning strategy for each database.
+A _partition_ is a logical grouping of data stored in [Apache Parquet](https://parquet.apache.org/)
+format in the InfluxDB v3 storage engine. By default, data is partitioned by day,
+but, depending on your schema and workload, customizing the partitioning
+strategy can improve query performance.
+
+Use the `--template-tag`, `--template-tag-bucket`, and `--template-timeformat`
+flags to define partition template parts used to generate partition keys for the database.
+
+For more information, see [Manage data partitioning](/influxdb/cloud-dedicated/admin/custom-partitions/).
+
+{{% note %}}
+#### Partition templates can only be applied on create
+
+You can only apply a partition template when creating a database.
+There is no way to update a partition template on an existing database.
+{{% /note %}}
+
+<!-------------------------------- END INFLUXCTL ------------------------------>
+{{% /tab-content %}}
+{{% tab-content %}}
+<!------------------------------- BEGIN cURL ---------------------------------->
+
+_This example uses [cURL](https://curl.se/) to send a Management HTTP API request, but you can use any HTTP client._
+
+1. If you haven't already, follow the instructions to [install cURL](https://everything.curl.dev/install/index.html) for your system.
+2. In your terminal, use cURL to send a request to the following endpoint:
+
+   {{% api-endpoint endpoint="https://{{< influxdb/host >}}/api/v0/accounts/ACCOUNT_ID/clusters/CLUSTER_ID/databases" method="post" api-ref="/influxdb/cloud-dedicated/api/management/#operation/CreateClusterDatabase" %}}
```

Comment:
Does this resolve correctly? Isn't the host for the management API a specific host?

Reply:
Right, they should be `https://console.influxdata.com/`.